Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -499,7 +499,7 @@ void copy_container_content(
   RefToContainer ref_wrapper)
 {
   ref_wrapper.get().reserve(vec.size());
-  for(const T& t : vec)
+  for(const T t : vec)
```

Comment:
?

Reply:
The compiler warning about the boolean being passed by reference.

Reply:
Ok, it is outdated now because I have reverted these changes. It was discussed in PR #5760 and we agreed that it is not important.